engine/src/main/java/io/knotx/fragments/engine/FragmentEvent.java

```diff
@@ -115,7 +118,19 @@ public String toString() {
   }
 
   public enum Status {
-    UNPROCESSED, SUCCESS, FAILURE
+    UNPROCESSED(null),
```

Comment:
I don't like null here. there should be a better way of doing this. maybe optional is better